polaris_wyre/wyre/integration.py

```diff
@@ -0,0 +1,147 @@
+from decimal import Decimal
+
+from polaris import settings as polaris_settings
+from polaris_wyre.wyre.dtos import TransferData
+from polaris.models import Asset, Transaction
+from polaris.integrations import CustodyIntegration
+from rest_framework.request import Request
+from stellar_sdk.server import Server
+
+from . import Wyre
+from .api import TEST_BASE_URL
+
+
+class WyreIntegration(CustodyIntegration):
+    def __init__(
+        self, api_token: str = "", account_id: str = "", api_url: str = TEST_BASE_URL
```

Comment:
Since `WyreAPI`, `Wyre` and `WyreIntegration` have the same constructor signature, I think it would be wise to start thinking about some creational pattern to avoid the complexity that this repetition will probably create in the future, and to also have dependency injection instead of creating the object inside the constructor. Let's talk about this some other time.